openshift_tools/ircbot/openshift_sre/openshift_sre.py

```diff
@@ -7,20 +7,20 @@
 Licensed under the MIT license.
 """
 from __future__ import print_function
-from datetime import datetime as dt
-from re import finditer, search
-from collections import OrderedDict
+from __future__ import unicode_literals
+from re import finditer
+import json
+import os
+import pypd
```

Comment:
I missed this. You need to add this to `openshift-tools/openshift_tools/ircbot/requirements.txt` file so it installs the dependency during the source-to-image build.